app/main.ipynb

```diff
@@ -56,33 +58,66 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "# write your code here (for Python default syntax)\n",
+    "a = [0] * 1000\n",
+    "sys.getsizeof(a)"
```

Comment:
The output of the list 'a' is printed, which is unnecessary for the task. Consider removing the print statement to avoid cluttering the output.